articles/modules/ROOT/pages/migrating-explicit-lucene-indexes-to-native-schema-indexes.adoc

```diff
@@ -0,0 +1,102 @@
+= Migrating Explicit Lucene Indexes to Native Schema Indexes
+:slug: migrating-explicit-lucene-indexes-to-native-schema-indexes
+:author: Ali Maddahian
+:category: operations
+:tags: lucene, index, legacy, explicit, capacity, schema, full-text
+:neo4j-versions: 1.x,2.x,3.x,4.x
+
+Given that there are still some customers on older releases still utilizing legacy/explicit indexes, we will point out a few pointers here on how to convert these indexes to native schema indexes when upgrading to Neo4j version 4.x, as legacy/explicit indexes have been deprecated as of 3.5.x, and totally removed from the 4.x version.
```

Comment:
"still" and "point" used twice each in the same sentence. Suggesting replacement/omission if possible.